website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,385 @@
+
+:::tip Your SSF Config URL is now available
+
+Assigning an SSF provider to a application will generate **SSF Config URL**.
+
+Navigate to **Providers** -> **Apple Business Manager SSF** and copy the `SSF Config URL` value to your text editor.
+
+:::
+
+---
+
+## Configuration checklist
+
+Let's recap the steps you've taken to configure authentik as an identity provider for Apple Business Manager:
+
+1. Created scope mappings:
+    - [x] User's profile information
+    - [x] read access
+    - [x] Management access.
+2. Created a OIDC provider to handle the authentication flow.
+    - [x] Added the scope mappings to the provider, **as well as the default `profile` scope.**
+    - [x] Specified the `Redirect URIs/Origins` and `Signing Key`.
+    - [x] Noted the `Client ID`, `Client Secret`, and `OpenID Configuration URL` in your text editor.
+3. Created a Shared Signals Framework provider to handle the backchannel communication between authentik and Apple Business Manager.
+    - [x] Specified the `Signing Key` and `Event Retention`.
+4. Created an application and assigned the SSF provider as a backchannel provider.
+    - [x] Noted the `SSF Config URL` in your text editor.
+
+:::tip Port numbers required
+
+By default, authentik will generate the **SSF Config URL** and **OpenID Configuration URL** without the default HTTPS port (443). However, Apple requires the port number to be included when providing the URLs in the configuration.
+
+From your text editor, update the URLs to include the port number `443`:
+
+#### 1. Add port to SSF Config URL
+
+```diff
+-https://authentik.company/.well-known/ssf-configuration/abm
++https://authentik.company:443/.well-known/ssf-configuration/abm
+```
+
+#### 2. Add port to OpenID Config URL
+
+```diff
+-https://authentik.company/application/o/abm/.well-known/openid-configuration
++https://authentik.company:443/application/o/abm/.well-known/openid-configuration
+```
+
+**This is a common source of configuration issues!**
+
+:::
+
+---
+
+## Apple Business Manager configuration
+
+With these prerequisites in place, authentik is ready to act as an identity provider for Managed Apple Accounts.
+
+:::info Managed Apple Accounts
+
+Similar to a personal Apple account, a _Managed Apple Account_ uses an email address to access Apple services and devices. What makes an Apple account _managed_ is that the domain associated with the email address is owned and verified by an organization through an Apple Business Manager account.
+
+By verifying the domain, Apple Business Manager will delegate ownership of any accounts with a matching email address to the organization, allowing for centralized management of devices, apps, and services.
+
+:::
+
+### 1. Add and verify your domain
+
+From the [Apple Business Manager dashboard](https://business.apple.com/), click **your account name** on the sidebar, then select **Preferences**. From the preferences page, select **Managed Apple Accounts** tab, and click **Add Domain**.
```

Comment:
Ideally, these are the steps that need to be numbered, so that readers staring at the ABM UI can walk through the steps easily, and recognize the exact fields in the UI. And, ideally, the steps (which follow the UI) are not broken up with the H4s like the 1.1 or 1.2 below.

Let's try to make this match the "authentik configuration" section (as far as numbering and following the UI) as much as possible, but... it doesn't have to be perfect. Let's just et this massive Guide merged. ;-)